tests/data/nlcd/data.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+
+import numpy as np
+import rasterio
+
+SIZE = 32
+
+np.random.seed(0)
+
+dir = "nlcd_2019_land_cover_l48_20210604"
+
+
+def create_file(path: str, dtype: str):
+    """Create the testing file."""
+    profile = {}
```

Comment:
This whole thing could be a dict created at a single time instead of constructing line by line, see the L7 Irish data.py for an example